style.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+h1 {
+  font-size: 20px;
+  color: #172b4d;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  margin: 130px 0 10px 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin-right: 100px;
+  padding-right: 100px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #172b4d;
+}
```

Comment:
- [ ] Dear @Dorcas126, Because you have set the margin and padding for your body, Your content I only taking the rest of the page. Please remember we need to phase spacings as illustrated on the figma design. 


https://user-images.githubusercontent.com/69077061/219342817-e52826b7-da75-4c5c-9520-943049ecceb4.mp4



Your design | Figma Design
-- | --
 
![Screenshot 2023-02-16 134351](https://user-images.githubusercontent.com/69077061/219343407-394b0495-1ab7-4010-9a68-c0170e8b1c4f.png) | ![219320208-662e4f97-5851-4c2a-9ddc-30cd24b6d323](https://user-images.githubusercontent.com/69077061/219342926-68a22ac4-f6ee-49f1-ad28-9501bc3a3785.png)